controls/views.py

```diff
@@ -2400,3 +2402,114 @@ def poam_export(request, system_id, format='xlsx'):
     else:
         # User does not have permission to this system
         raise Http404
+
+def project_import(request, project_id):
+    """
+    Import an entire project's components and control content
+    """
+    project = Project.objects.get(id=project_id)
```

Comment:
I guess I'm confused why a project_id is even required for this function, and where it gets passed in from. I can see that the template passes it in, but how would that template have access to the project id before the project has been imported? I assume that project id can be parsed out from the json, so it probably has to be done after the json is loaded in this function, but not before.

Reply:
@davidpofo Thanks for answering this. I think long-term this may benefit from being refactored to not require a project_id (in case we allow project imports from outside of a project/system), but for now this works.